src/main/java/spoon/support/adaption/NodePrintHelper.java

```diff
@@ -0,0 +1,39 @@
+package spoon.support.adaption;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+final class NodePrintHelper {
+
+	private NodePrintHelper() {
+		throw new AssertionError("No instantiation");
+	}
+
+	/**
+	 * Encloses a string in double quotes. Does not escape inner quotation marks.
+	 * Special-purpose helper for {@link DeclarationNode} and {@link GlueNode}.
+	 *
+	 * @param input the string to quote
+	 * @return the string surrounded with quotes
+	 */
+	public static String quote(String input) {
+		return '"' + input + '"';
+	}
+
+	/**
+	 * Converts a list of values to a crudely formatted JSON-y array string.
+	 * Special-purpose helper for {@link DeclarationNode} and {@link GlueNode}.
+	 *
+	 * @param input the input list
+	 * @return the list as a quoted array
+	 */
+	public static String quote(List<?> input) {
```

Comment:
The method name is not sufficiently descriptive imo, and I generally think that overloads should be used exceedingly sparingly. Ought to be something like:

```suggestion
	public static String toJSONArray(List<?> input) {
```

or

```suggestion
	public static String toStringifiedJSONArray(List<?> input) {
```

Reply:
It isn't a general-purpose Json-ify method, as it does not handle embedded quotes and control char escaping and so on. It just wraps quotes around the input which is enough to turn class names into valid Json. I only made it mostly Json-like so editors can neatly fold, highlight and organize the output.

I've renamed it and added a further disclaimer in the doc.